### DISCUSSION TITLE
post/linq-challenge-1-answers-in-fsharp

### DISCUSSION THREAD

**Discussion by giscus** [markheath/soundcode-blog-comments#55] [Announcements]
# post/linq-challenge-1-answers-in-fsharp

Mark Heath's Development Blog

https://markheath.net/post/linq-challenge-1-answers-in-fsharp

**Comment by djcarter85** [comment]
Here's how I did question 4: https://gist.github.com/djcarter85/8e4d84379e63c341366fc1fd470ab97f

Instead of yielding all of a sub sequence, I just decided to loop through both sequence and yield individual values. I think it looks a bit more like the query syntax version in C# and is a bit neater.

**Reply by markheath** [reply]
very nice, I like it
